src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/repositories/ReportedAdjudicationRepository.kt

```diff
@@ -33,11 +33,19 @@ interface ReportedAdjudicationRepository : CrudRepository<ReportedAdjudication,
     value = "select * from reported_adjudications ra " +
       "where ra.date_time_of_discovery > :startDate and ra.date_time_of_discovery <= :endDate " +
```

Comment:
Minor one - we could extract the SQL starting from
`where ra.date_time_of_discovery > :startDate and ra.date_time_of_discovery...`
onwards into a constant and reference it from `value` and `countQuery` so fewer things to update?